src/wad/tex.rs

```diff
@@ -264,23 +276,27 @@ impl TextureDirectory {
         let mut offsets = BTreeMap::new();
         let mut data = vec![255u8; width * height];
         let (mut row, mut column) = (0, 0);
-        info!("Flat atlas size: {}x{} ({}, {})", width, height, flats_per_row,
-                                                 num_rows);
+        info!("Flat atlas size: {}x{} ({}, {})",
+              width,
+              height,
+              flats_per_row,
+              num_rows);
```

Comment:
I'm OK with leaving this as is, but in general, I think rustfmt needs a different approach to `format`/`info` arguments. Maybe treat them as if they were in a tuple: then all format args could go on the same line after the format string? Just a thought.